custom/icds/urls.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.conf.urls import url
+from custom.icds.views.ccz_hosting import (
+    ManageCCZHosting,
+    ManageCCZHostingLink,
+    EditCCZHostingLink,
+    CCZHostingView,
+    remove_ccz_hosting,
+    download_ccz,
+    download_ccz_supporting_files,
+)
+
+urlpatterns = [
+    url(r'^ccz/hostings/manage', ManageCCZHosting.as_view(), name=ManageCCZHosting.urlname),
+    url(r'^ccz/hostings/links', ManageCCZHostingLink.as_view(), name=ManageCCZHostingLink.urlname),
+    url(r'^ccz/hostings/link/(?P<link_id>[\d-]+)/edit/', EditCCZHostingLink.as_view(),
+        name=EditCCZHostingLink.urlname),
+    url(r'^ccz/hostings/link/(?P<hosting_id>[\d-]+)/delete/', remove_ccz_hosting,
+        name="remove_ccz_hosting"),
+    url(r'^ccz/hostings/(?P<hosting_id>[\w-]+)/download/(?P<blob_id>[\w-]+)/', download_ccz,
```

Comment:
Sounds good. i see that blob id is not needed even now since we have the hosting_id. Though it's needed for the supporting docs/files.
(yet to look at the other comment)

Reply:
The blob id was removed from the url and the view. Just hosting id was sufficient